src/applications/personalization/dashboard/components/debts-v2/DebtsV2.jsx

```diff
@@ -136,11 +162,11 @@ const BenefitPaymentsAndDebtV2 = ({
             {copaysCount > 0 && (
               <>
                 <DashboardWidgetWrapper>
-                  {hasDebtError && <OutstandingDebtsError />}
+                  {/* {!hasCopayError && <OutstandingDebtsError />} */}

```

Comment:
Can this just be removed since it looks like it is rendered conditionally above?